content/en/real_user_monitoring/guide/session-replay-getting-started.md

```diff
@@ -0,0 +1,161 @@
+
+``` html
+<div id=”profile-info” data-dd-privacy=”hidden”>
+    <p>Name: John Doe</p>
+    <p>Birth date: June 6th, 1987</p>
+</div>
+```
+
+## Troubleshooting
+
+### Some HTML elements are not visible at replay
+
+Session Replay does not support the following HTML elements at the moment: `iframe`, `video`, `audio`, `canvas`, as well as Web Components.
+
+### Fonts or images do not render properly
+
+A Session Replay is not video, but an actual iframe rebuilt based on snapshots of the DOM. The replay thus depends on the various assets of the page: fonts & images.
+
+Several reasons might explain why assets are not available at the time of the replay.
+
+1. The resource does not exist anymore. For example, it was part of a previous deployment.
+2. The resource is inaccessible. For example, authentication might be required, or the resource might only be accessible from an internal network.
+3. The resource is blocked by the browser due to CORS (typically, web-fonts). 
+The replay being rendered on the `app.datadoghq.com` domain, the assets requests are subject to cross origin security checks by your browser. If the given asset is not authorised for the domain, your browser blocks the request.
+The fix is thus to allow `app.datadoghq.com through` the `[Access-Control-Allow-Origin][5]` header for any font or image assets your website depends upon, ensuring these resources are accessible for the replay.
+To learn more about Cross Origin Resource Sharing, see the [MDN Web Docs article][6].
+
+### CSS rules not properly applied / mouse hover not replayed 
+
+Contrary to fonts & images, the recorder will try to bundle the various CSS rules applied as part of the recording data, leveraging the [CSSStyleSheet][7] interface. If this is not possible, it will fallback to recording the links to the CSS files.
+
+Note that for proper mouse hover support, the CSS rules must be accessible via the CSSStyleSheet interface.
+
+If the stylesheets are hosted on a different domain than the web page, access to the CSS rules is subject to cross origin security checks by the browser and the browser must be instructed to load the stylesheet leveraging CORS via the [crossorigin][8] attribute.
+
+For example, if your application is on the `example.com` domain and depends on a CSS file on `assets.example.com` via a link element, the `crossoriring` attribute should be set to `anonymous`, unless credentials are required:
+
+``` html
+<link rel="stylesheet" crossorigin="anonymous"
+      href="https://assets.example.com/style.css”>
+```
+
+Additionally, the `example.com` domain must be authorised by `assets.example.com` to load it’s resource by properly setting the `[Access-Control-Allow-Origin][5]` header.
+
+## Frequently asked questions
```

Comment:
We don't have FAQs in docs pages because we find that it has poor SEO, instead we should add this content where it makes the most sense in the rest of the document. The "How it works" section should be moved up under the introduction "What is Session Replay?" section. The performance impact could also be added into the introduction, and then the availability section can be added to the configuration section.

Reply:
This is not a page that will be indexed, we'll integrate Session Replay docs into the full docs once it's GA